www/main/navigator/shabad/utils/change-verse.js

```diff
@@ -119,11 +119,18 @@ const skipMangla = (shabadVerses, index) => {
 
 export const intelligentNextVerse = (
   filteredItems,
-  { activeVerseId, previousVerseIndex, setPreviousIndex, atHome, setHome, homeVerse },
+  {
+    activeVerseId,
+    previousVerseIndex,
+    setPreviousIndex,
+    atHome,
+    setHome,
+    homeVerse,
+    intelligentSpacebar,
+  },
 ) => {
-  if (homeVerse) {
-    const currentVerseIndex = filteredItems.findIndex(({ verseId }) => verseId === activeVerseId);
-    let nextVerseIndex;
+  function handleIntelligentSpacebar(nextIndex, currentVerseIndex) {
```

Comment:
Can you change this to ES6 arrow body function, to keep it consistent with the other code.